src/go-postgres/devcontainer-template.json

```
@@ -1,30 +1,27 @@
 {
     "id": "go-postgres",
-    "version": "1.2.2",
+    "version": "2.0.0",
     "name": "Go & PostgreSQL",
     "description": "Use and develop Go + Postgres applications. Includes appropriate runtime args, Go, common tools, extensions, and dependencies.",
     "documentationURL": "https://github.com/devcontainers/templates/tree/main/src/go-postgres",
     "publisher": "Dev Container Spec Maintainers",
     "licenseURL": "https://github.com/devcontainers/templates/blob/main/LICENSE",
     "options": {
-        "imageVariant": {
+		"imageVariant": {
             "type": "string",
-            "description": "Go version (use -bullseye variants on local arm64/Apple Silicon):",
+            "description": "Go version (use -bookworm, or -bullseye variants on local arm64/Apple Silicon):",
```

I wonder if we should adopt a pattern in these descriptions to quickly explain the OS bit.  Something like

```suggestion
            "description": "Go version (following the pattern `{go-version}-{os-version}`. Use -bookworm, or -bullseye variants on local arm64/Apple Silicon):",
```